docs/administrator-guide/configuration/others/README.md

```diff
@@ -162,21 +164,21 @@ Changing the timezone will mostly affect:
 ## Anonymous usage report
 
 Understanding how you use Kestra is very important to us: it helps us improve the solution in many ways.
-For this very reason, the `kestra.anonymous-usage-report.enabled` option is mandatory: we want you to take time to consider whether you wish to share anonymous data with us, so we can benefit from your experience and use cases.
+For this reason, the `kestra.anonymous-usage-report.enabled` option is mandatory: we want you to take time to consider whether you wish to share anonymous data with us, so we can benefit from your experience and use cases.
```

Comment:
```suggestion
For this reason, the `kestra.anonymous-usage-report.enabled` option is mandatory: we want you to consider whether you wish to share anonymous data with us so that we can benefit from your experience and use cases.
```